README.adoc

```diff
@@ -11,29 +11,29 @@
 //
 :page-layout: guide
 // INSTRUCTION: The project id is the part of the git repository after the guide- and must be specified
-:projectid: github repo name without the guide- prefix
+:projectid: outline
 // INSTRUCTION: Provide an estimate of how long the guide will take to go through.
 :page-duration: 15 minutes
 // INSTRUCTION: Provide the date when the guide is published.  Format is YYYY-MM-DD.
-:page-releasedate: 2017-11-17
+:page-releasedate: 2018-05-14
 // INSTRUCTION: Provide a description for the guide index page.
-:page-description: Learn how to create a REST service with JAX-RS, JSON-P, and Open Liberty.
+:page-description: Learn how to create a todo list API as a REST service using JAX-RS, CDI, and Open Liberty.
 // INSTUCTION: Please provide relevant tags, try to avoid inventing new ones, tags where there is 1 guide for the tag isn't useful.
-:page-tags: ['REST', 'Getting Started']
+:page-tags: ['REST', 'Todo']
```

Comment:
I think the tag should be "Getting Started" instead of "Todo", as "Todo" is not meaningful for searching. Also "Open Liberty", "CDI", "JAX-RS" can be other tags too.